docs/experience/url-groups.md

```diff
@@ -0,0 +1,39 @@
+title: URL Groups
+description: Guide for using URL Groups in Experience
+
+Most websites follow certain URL patterns for their pages. For example, your website might show user profiles at `/user/123` or `/user/312`. Therefore, the URL pattern is `/user/{userId}`. Mapping these patterns with URL Groups in Sematext Experience can give you a better overview of the performance data for each part of your website. The idea is that all user profiles will have more or less the same performance characteristics, so it makes sense to monitor the "User Profile" page as a whole instead of monitoring individual user profile pages.
+
+If we keep in mind the above example, here are the differences you will see if you use Url Groups.
```

Comment:
If we keep in mind the above example, here are the differences you will see if you use Url Groups.
->
If we keep in mind the example above, here are the differences you will see if you use URL Groups.